server/start.go

```diff
@@ -385,7 +389,9 @@ func setupNodeAndExecutor(
 			return nil, nil, cleanupFn, err
 		}
 
-		rollkitGenesis = createEvolveGenesisFromCometBFT(cmtGenDoc)
+		evGenesis = createEvolveGenesisFromCometBFT(cmtGenDoc)
```

Comment:
can we add DAStartHeight as an extra arg so the evGenesis is fully created in one fn call without needing to modify it again after.